inlong-agent/agent-common/src/main/java/org/apache/inlong/agent/constant/JobConstants.java

```diff
@@ -43,7 +43,7 @@
 
     //File job
     public static final String JOB_TRIGGER = "job.fileJob.trigger";
-    public static final String JOB_LINE_FILTER_PATTERN = "job.fileJob.dir.pattern";
+    public static final String JOB_LINE_FILTER_PATTERN = "job.fileJob.pattern";
```

Comment:
this param is used in kafka job also, should not use filejob prefix

Reply:
done